Server/VikingVault.Services/UserService.cs

```diff
@@ -122,5 +122,29 @@ public void DeleteUser(UserEmail userEmail)
                 throw new DatabaseException(e.Message);
             }
         }
+
+        public UpdateUserDTO UpdateUser(UpdateUserDTO user)
+        {
+            try
+            {
+                User userToUpdate = _dbContext.User.SingleOrDefault(u => u.Email == user.Email);
+                if(userToUpdate == null)
+                {
+                    throw new UserServiceException("User not found");
+                }
+                userToUpdate.FirstName = user.FirstName;
```

Comment:
You can use object initializers in here for a clearer syntax, https://www.c-sharpcorner.com/UploadFile/mahesh/object-initializer-in-C-Sharp/

Reply:
But I don't need to create a new User, just update some of its properties